### ISSUE TITLE
CLI Command Line Arguments

### ISSUE THREAD

**Issue by Lvl4Sword** [Bitmessage/PyBitmessage#1035]
Daemon and Portable Mode should be command line arguments. This allows other programs to use the CLI much easier.

**Comment by sp2004c** [comment]
true, one can set BITMESSAGE_HOME , does not even have to be an extra line in a bash script. Same effect.
